src/main/java/com/gooddata/md/AbstractObj.java

```diff
@@ -20,12 +20,16 @@
 public abstract class AbstractObj {
 
     @JsonProperty("meta")
-    protected final Meta meta;
+    protected Meta meta;
 
     protected AbstractObj(@JsonProperty("meta") Meta meta) {
         this.meta = meta;
     }
 
+    protected AbstractObj() {
+        this.meta = null;
```

Comment:
which `AbstractObj` has no meta? And is it really kind of `AbstractObj` then?

Reply:
well agreed on making `AbstractObj implements Serializable` - please create separate issue and PR